src/Tribe/RSVP.php

```diff
@@ -890,12 +885,13 @@ public function create_attendee_for_ticket( $ticket, $attendee_data ) {
 			$data['user_id'] = $user_id;
 		}
 
-		$repository->set_args( $data );
+		$attendee_object = tribe( 'tickets.attendees' )->create_attendee( $ticket, $data );
 
-		$attendee_id = $repository->create();
+		if ( ! $attendee_object ) {
+			throw new Exception( __( 'Unable to process your request, attendee creation failed.', 'event-tickets' ) );
+		}
 
-		// Update this after attendee is created.
-		update_post_meta( $attendee_id, $this->security_code, $this->generate_security_code( $attendee_id ) );
+		$attendee_id     = $attendee_object->ID;
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 5 spaces

<sub>hash: 3fbc76c902e8e554b200a003b8d12f8c</sub>